acrpull/pipeline.yaml

```diff
@@ -16,3 +16,6 @@ resourceGroups:
         variables:
           - name: ACRPULL_DIGEST
             configRef: acrPullImageDigest
+      - name: undeploy
+        action: Shell
+        command: make undeploy
```

Comment:
a pipeline.yaml file is used in MSFTs EV2 to deploy to production and executes all steps one after the other.

by adding an undeploy step after the deploy step, this pipeline would rollout and remove ACR pull